default.nix

```diff
@@ -38,10 +41,10 @@ in {
              (compiler.callPackage ./iohk/default.nix {})
              (drv: {
                 executableToolDepends = [ pkgs.makeWrapper ];
-                libraryHaskellDepends = iohk-ops-extra-runtime-deps;
+                libraryHaskellDepends = iohk-ops-extra-runtime-deps ++ [ nixops ];
```

Comment:
shouldn't nixops be part of `iohk-ops-extra-runtime-deps` as well?

Reply:
Done..